crates/cli/src/main.rs

```diff
@@ -0,0 +1,150 @@
+use clap::Parser;
+use clap::Subcommand;
+use gevulot_node::rpc_client::RpcClient;
+use std::net::SocketAddr;
+use std::path::PathBuf;
+
+#[derive(Parser, Debug)]
+#[clap(author = "Gevulot Team", version, about, long_about = None)]
+pub struct ArgConfiguration {
+    /// RPC url of the Gevulot node
+    #[clap(
+        short,
+        long,
+        default_value = "http://localhost:9944",
+        value_name = "URL"
+    )]
+    jsonurl: String,
+    /// Private key file path to sign Tx.
+    #[clap(
+        short,
+        long,
+        default_value = "localkey.pki",
+        value_name = "KEY FILE PATH"
+    )]
+    keyfile: PathBuf,
+    #[command(subcommand)]
+    command: ConfCommands,
+}
+
+#[derive(Subcommand, Debug)]
+enum ConfCommands {
+    /// Deploy prover and verifier.
+    GenerateKey,
+
+    /// Deploy prover and verifier.
+    #[command(arg_required_else_help = true)]
+    Deploy {
+        /// name of the deployment.
+        #[clap(short, long, value_name = "DEPLOYMENT NAME")]
+        name: String,
+        /// file path containing the img of the prover to deploy or the hash  of the prover img (--proverimgurl is mandorty in this case). If the  file is not found the parameters is used as an hash.
+        #[clap(short, long, value_name = "PROVER FILE or HASH")]
+        prover: String,
+        /// file path containing the img of the verifier to deployor the hash  of the verifier img (--verifierimgurl is mandorty in this case). If the  file is not found the parameters is used as an hash.
+        #[clap(short, long, value_name = "VERIFIER FILE or HASH")]
+        verifier: String,
+        /// url to get the prover img. If provided the prover will use this url to get the prover img. If not the cli tool start a local HTTP server to server the file to the node.
+        #[clap(long, value_name = "PROVER URL")]
+        proverimgurl: Option<String>,
+        /// url to get the verifier img. If provided the verifier will use this url to get the verifier img. If not the cli tool start a local HTTP server to server the file to the node.
+        #[clap(long, value_name = "VERIFIER URL")]
+        verifierimgurl: Option<String>,
```

Comment:
```suggestion
        verifier_img_url: Option<String>,
```
Like above & elsewhere, WDYT?

Reply:
I'll do a find replace in the code and push the change.